solnlib/bulletin_rest_client.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from solnlib import splunk_rest_client as rest_client
+from splunklib import binding
+import json
+
+
+__all__ = ["BulletinRestClient"]
+
+
+class BulletinRestClient:
+    READ_ENDPOINT = "/services/admin/messages"
+    CREATE_DELETE_ENDPOINT = "/services/messages"
+
+    headers = [("Content-Type", "application/json")]
+
+    def __init__(
+        self,
+        message_name: str,
+        session_key: str,
+        logger: logging.Logger = None,
```

Comment:
Optional[logging.Logger], we don't have a mypy here but it would complain :)

Reply:
yes, sorry for the confusion but I had a git problem this morning and it pushed only my first, working commit. At the moment I have fixed the issue with Git and this is a PR that should be reviewed.